testing/environments/snapshot.yml

```diff
@@ -17,7 +17,7 @@ services:
     - "indices.id_field_data.enabled=true"
 
   logstash:
-    image: docker.elastic.co/logstash/logstash:8.0.0-SNAPSHOT
```

Comment:
The image is normally maintained by RE. @mikemadden42 please create a follow up PR restoring 8.0.0-SNAPSHOT. We can retrigger CI every now and then.